core/src/test/java/nl/nn/adapterframework/parameters/ParameterTest.java

```diff
@@ -1088,9 +1088,9 @@ public void testFixedDate() throws Exception {
 			Object result = p.getValue(alreadyResolvedParameters, message, session, false); //Should return PutSystemDateInSession.FIXEDDATETIME
 			assertTrue(result instanceof Date);
 
+
 			Date resultDate = (Date) result;
-			SimpleDateFormat sdf = new SimpleDateFormat(Parameter.TYPE_DATE_PATTERN);
-			String formattedDate = sdf.format(resultDate);
+			String formattedDate = DateUtils.format(resultDate, Parameter.TYPE_DATE_PATTERN);
```

Comment:
Het zou handiger zijn om 1 formatter te hebben. Misschien behoorlijk buiten scope om dit overal te doen, maar probeer er wel op te letten. (Zeker omdat dit een test is.)